dartagnan/src/test/java/com/dat3m/dartagnan/AbstractDartagnanTest.java

```diff
@@ -64,19 +71,29 @@
         this.settings = settings;
     }
 
-    @Test
+    private void initSolverContext() throws Exception {
```

Comment:
This should return the Context instead of assigning to some member variable since the context is used only locally anyway.